content/chainguard/chainctl-usage/chainctl-version-update.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Find and Update Your chainctl Release Version"
+lead: ""
+description: "chainctl version basics"
+type: "article"
+date: 2025-03-06T08:49:15+00:00
+lastmod: 2025-03-06T08:49:15+00:00
+draft: false
+tags: ["chainctl", "version", "update", "Product"]
+images: []
+weight: 100
+---
+
+This page shows you how to check which release version of `chainctl` you have installed and how to update to the latest release.
+
+For a full reference of all commands with details and switches, see [chainctl Reference](/chainguard/chainctl/).
+
+
+## View your chainctl version
+
+To see which `chainctl` version you have installed, use:
+
+```shell
+chainctl version
+```
+
+This command tells you more than just a release number. The release is shown here as `GitVersion`.
+
+
+```shell
+
+               ▄▄▄▄▄▄▄▄▄▄▄              
+             ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄            
+           ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄          
+          ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄         
+         ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄        
+        ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄        
+        ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄       
+        ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄       
+        ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄       
+        ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄       
+     ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄    
+  ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄ 
+ ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+ ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+  ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄   ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄ 
+           ▄▄▄▄▄▄▄     ▄▄▄▄▄▄▄▄         
+
+   ____   _   _      _      ___   _   _    ____   _____   _
+  / ___| | | | |    / \    |_ _| | \ | |  / ___| |_   _| | |
+ | |     | |_| |   / _ \    | |  |  \| | | |       | |   | |
+ | |___  |  _  |  / ___ \   | |  | |\  | | |___    | |   | |___
+  \____| |_| |_| /_/   \_\ |___| |_| \_|  \____|   |_|   |_____|
+chainctl: Chainguard Control
+
+GitVersion:    v0.2.66
+GitCommit:     cb0ff7f1806341b20fd72be4769668a1985ed845
+GitTreeState:  clean
+BuildDate:     2025-04-09T07:49:51Z
+GoVersion:     go1.24.1
+Compiler:      gc
+Platform:      linux/amd64
+
+```
+
+
+
+
+## Update your chainctl install
+
+To update your `chainctl` installation to the latest release, use:
+
+```shell
+chainctl update
+```
+
+Updating requires administrative privileges, so be prepared to enter your machine's admin password.
+
+```shell
+sudo chainctl update
```

Comment:
Is there a need here for a note here about running it in `sudo` mode rather than the above example?

Reply:
The sentence above says why .. but in my experience thats not true...